src/host/cmdline.h

```diff
@@ -158,7 +158,7 @@ void RedrawCommandLine(COOKED_READ_DATA& cookedReadData);
 #define WC_DELAY_EOL_WRAP        0x80
 
 // Word delimiters
-bool IsWordDelim(const WCHAR wch);
+bool IsWordDelim(const wchar_t wch);

```

Comment:
IntelliSense warning about function not implemented due to wchar_t vs. WCHAR inconsistency.

Reply:
Thanks for switching it to `wchar_t`, it turns out that the Windows coding guidelines prefer built in C++ types over Windows ones when it makes sense.